doc/README.md

```diff
@@ -0,0 +1,39 @@
+# Documentation
```

Comment:
As with previous file, please move to root. We can think of how to integrate that with the github-pages, but for now, let's leave it at the top level to avoid confusion.